analytics/management/commands/populate_analytics_db.py

```diff
@@ -104,7 +104,9 @@ def handle(self, *args: Any, **options: Any) -> None:
         Subscription.objects.bulk_create(subs)
 
         def insert_fixture_data(
-            stat: CountStat, fixture_data: Mapping[Optional[str], List[int]], table: Type[BaseCount]
+            stat: CountStat,
+            fixture_data: Mapping[Union[str, int, None], List[int]],
```

Comment:
This might be better done by defining a `CountData` type rather than repeating it N times?  Not sure, if you agree feel free to add as a follow-up commit.

Reply:
Sure